libs/langgraph/langgraph/prebuilt/tool_node.py

```diff
@@ -112,16 +101,13 @@ async def _afunc(
 
         async def run_one(call: ToolCall):
             try:
-                output = await self.tools_by_name[call["name"]].ainvoke(
-                    call["args"], config
-                )
+                input = {**call, **{"type": "tool_call"}}
+                return await self.tools_by_name[call["name"]].ainvoke(input, config)
             except Exception as e:
                 if not self.handle_tool_errors:
                     raise e
-                output = f"Error: {repr(e)}\n Please fix your mistakes."
-            return ToolMessage(
-                content=str_output(output), name=call["name"], tool_call_id=call["id"]
-            )
+                content = f"Error: {repr(e)}\n Please fix your mistakes."
+                return ToolMessage(content, name=call["name"], tool_call_id=call["id"])
```

Comment:
why are we no lnoger passing content by keyword?

Reply:
cleaner this way (imo), messages accept content as pos arg